commercial/app/views/hosted/guardianHostedGallery.scala.html

```diff
@@ -42,72 +47,72 @@ <h3 class="hosted-gallery__sub-heading">{page.standfirst}</h3>
+                                    target="_blank"
+                                    title="twitter">
+                                        <span class='inline-icon__fallback button share-modal__item share-modal__item--twitter'>Share on Twitter</span>
+                                        @fragments.inlineSvg("share-twitter", "icon", List("rounded-icon social-icon social-icon--twitter"))
+                                    </a>
+                                </li>
+                                <li class="social__item social__item--email" data-link-name="email">
+                                    <a class="social__action social-icon-wrapper"
+                                    data-link-name="email-social-share-1"
+                                    href="mailto:?subject=@{page.pageTitle}&body=@{page.pageUrl}"
+                                    target="_blank"
+                                    title="email">
+                                        <span class='inline-icon__fallback button share-modal__item share-modal__item--email'>Share via email</span>
+                                        @fragments.inlineSvg("share-email", "icon", List("rounded-icon social-icon social-icon--email"))
+                                    </a>
+                                </li>
+                                <li class="social__item social__item--pinterest" data-link-name="pinterest">
+                                    <a  class="social__action social-icon-wrapper"
+                                    data-link-name="Pinterest-social-share-1"
+                                    href="https://www.pinterest.com/pin/find/?url=@{page.pageUrl}"
+                                    target="_blank"
+                                    title="pinterest">
+                                        <span class='inline-icon__fallback button share-modal__item share-modal__item--pinterest'>Share on Pinterest</span>
+                                        @fragments.inlineSvg("share-pinterest", "icon", List("rounded-icon social-icon social-icon--pinterest"))
+                                    </a>
+                                </li>
+                            </ul>
+                        </div>
+                    </header>
+                    <div class="hosted-gallery__captions">
+                        @for(i <- page.images.indices) {
+                            <div class="hosted-gallery__caption js-hosted-gallery-caption @{if (i == 0) "current-caption" else ""}">
+                                @fragments.inlineSvg("camera", "icon", List("inline-camera inline-icon "))
+                                <span>@page.images(i).title</span>
+                                <div class="hosted-gallery__caption-text">
+                                    @page.images(i).caption
+                                    @{if (page.images(i).caption.length > 0 && page.images(i).credit.length > 0) <br/>}
```

Comment:
Could be
`@if (page.images(i).caption.nonEmpty && page.images(i).credit.nonEmpty) {<br/>}`